setup.py

```diff
@@ -3,6 +3,8 @@
 import os
 import re
 import shutil
+import setuptools
```

Comment:
this is imported but i don't see it used on the commit. does it needed?

Reply:
As I mentioned in my Issue, I was building this on Windows 10 but couldn't package it as a .exe - I thought setuptools would help, but apparently not.